### ISSUE TITLE
`same as <token>` is not allowed.

### ISSUE THREAD

**Issue by RSickenberg** [VincentLanglet/Twig-CS-Fixer#111]
### Rule(s) related to or rule(s) missing

When I try `same as <token>` I have this error:

```
File is invalid: A hash value must be followed by a comma. Unexpected token "name" of value "active" ("punctuation"
          expected with value ",").
```

### Expected behavior

`active: attendance.response is same as state` should works

#### Context:
`attendance` is an object
`attendance.response` is an enum.

`state` is the same enum but in a loop of all cases.

### Actual behavior

`active: attendance.response is same as (state)` is working.

**Comment by VincentLanglet** [comment]
Looking at the Twig doc it's used with `()`
https://twig.symfony.com/doc/2.x/tests/sameas.html

**Comment by RSickenberg** [comment]
For the twig code is's used in an object.

```twig
{% set myObject = [{
	active: attendance.response is same as state
}] %}
```

Which worked as intended.

![Screenshot 2023-01-04 at 13 09 17](https://user-images.githubusercontent.com/16922548/210552311-1d62027b-775b-453b-9d35-55ebc0456dd2.png)
> Here is an example if I try to reproduce the issue.

However, I agree that the doc example is used with `()` which is perfectly fine for me to use.

**Comment by VincentLanglet** [comment]
I'll see if I can support the syntax.

**Comment by RSickenberg** [comment]
@VincentLanglet Many thanks!

Let me know if you may want some help!

**Comment by VincentLanglet** [comment]
Sure, @RSickenberg can you try this branch on your project ? https://github.com/VincentLanglet/Twig-CS-Fixer/pull/112

**Comment by VincentLanglet** [comment]
Thanks ; I just need to fix the lowest build.

**Comment by VincentLanglet** [comment]
Released in 0.8.1

**Comment by RSickenberg** [comment]
Merci Vincent ! 😄
